examples/pipeline/kg_builder_with_schema.py

```diff
@@ -0,0 +1,117 @@
+#  Copyright (c) "Neo4j"
```

Comment:
Is this different from the other kg_builder example?

Reply:
Apologies that was an old example. I've pushed the correct one here https://github.com/neo4j/neo4j-genai-python/pull/88/commits/c8f8ba5692855bf054bc6145fdb5de2aa1ebb4f7